docs/scos/dev/front-end-development/202311.0/oryx/how-to-contribute.md

```diff
@@ -0,0 +1,58 @@
+---
+title: How to contribute to the Oryx
+description: The guide describes how to contribute to the Oryx framework
+template: howto-guide-template
+---
+
+# Contributing to Oryx
+
+Welcome to the Oryx Framework Contribution Guide! We appreciate your interest in contributing to Oryx and becoming part of our community. 
+
+This guide provides essential information to help you get started with contributing to the development of the Oryx frontend framework. Whether you're a seasoned developer or just getting started, your contributions are valuable to us. In this guide, you'll find details on setting up your development environment, understanding our contribution process, and guidelines for submitting your contributions. 
+
+## Prerequisites
+
+- [Node.js](https://nodejs.org/) or a compatible Javascript runtime
+- npm package manager
+
+## Semantic Versioning
+
+Oryx follows semantic versioning. We release patch versions for critical bugfixes, minor versions for new features or non-essential changes, and major versions for any breaking changes. When we make breaking changes, we also introduce deprecation warnings in a minor version so that our users learn about the upcoming changes and migrate their code in advance. Learn more about it - [Oryx versioning](/docs/scos/dev/front-end-development/{{page.version}}/oryx/getting-started/oryx-versioning.html)
+
+## Branch Organization
+
+Submit all changes directly to the `development` branch. We do our best to keep `development` in a good shape, with all tests passing.
+Code that lands in main must be compatible with the latest minor release. It may contain additional features, but no breaking changes. 
+
+## Sending a Pull Request
+
+Oryx team is monitoring open pull requests. We will review your pull request and either merge it, request changes to it or close it with an explanation. We’ll do our best to provide updates and feedback throughout the process.
+
+How to send a pull request:
+
+- Fork [Oryx repository](https://github.com/spryker/oryx) and create your branch from main.
+- Run `npm install` in the repository root.  
+- If you’ve added code that should be tested, add tests.
+- Ensure all GitHub actions required checks are passed.
+- Provide a clear and concise description of your changes in the pull request description.
+
```

Comment:
Add information about what happens next. After approving a pull request we will update a reporter with the next steps. Pull request can be rejected, approved or we may ask for some clarification or changes.